backend/coins/btc/account.go

```diff
@@ -867,10 +866,6 @@ var defaultScriptType = "p2wpkh"
 //	#2: base64 encoding of the message signature, obtained using the private key linked to the address.
 //	#3: is an optional error that could be generated during the execution of the function.
 func SignBTCAddress(account accounts.Interface, message string, format string) (string, string, error) {
-	if !exchanges.IsPocketSupported(account) {
```

Comment:
This is not needed anymore, as we previously moved the endpoint that calls the function from handlers/handlers.go to coins/btc/handlers.go, so it will always be called on btc accounts. Also this is not exclusively used for Pocket anymore

Reply:
the function signature is still closely tied to pocket or the request-address NPM package:

```
`format` is the script type that should be used in the address derivation, as received by the widget
//		(see https://github.com/pocketbitcoin/request-address#requestaddressv0messagescripttype).
//		If format is empty, native segwit type is used as a fallback.
```

Probably too much overhead now, but maybe better if this function took `signing.ScriptType` directly and the conversion happened at the callsite. I'd just worry a bit that in the future there will be other uses of this function unrelated to the NPM package, in which case the stringly-typed format is not so great.

Reply:
Yeah, you're right. Since I was there, I added a second commit to refactor it. Now the frontend has an util that converts from the request-address library script type to the script type used in the rest of the app, and the backend doesn't depend on it anymore.